source/iris/download.html

```diff
@@ -4,13 +4,21 @@
 
 {% block content %}
 
-     <div id="bd">
-            <h1>Download</h1>
-            <h2>Official releases</h2>
-             <p>Official releases can be found <a href="https://github.com/SciTools/iris/tags">here</a>. These releases contain source, documentation and instructions on how to build and install Iris under Linux.</p>
-            <h2>GitHub repository</h2>
-             <p>We strongly encourage people to contribute to Iris and for this type of development we suggest you obtain the latest version of the code from our github repository. Assuming you have <a href="http:git-scm.org">git</a> installed, enter the following command into a terminal:</p>
-        </p><pre class="code">git clone https://github.com/SciTools/iris.git</pre>
```

Comment:
As above (except that this is the html source that's added to the final html page, which the comment is on above).